tailormap-components/projects/core/src/lib/analysis/state/analysis.effects.ts

```diff
@@ -32,7 +28,7 @@ export class AnalysisEffects {
   ));
 
   constructor(
-    private store$: Store<AnalysisState>,
+    private store$: Store,
```

Comment:
Because the masters say so: https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/docs/rules/no-typed-global-store.md

Not 100% sure myself yet, it could help identify / think about which feature state you want to use, even though it means nothing because global store is always injected